.github/workflows/diff.yml

```diff
@@ -0,0 +1,30 @@
+name: Generated code
+on: [push, pull_request]
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 10
```

Comment:
Do we really need `fetch-depth=10`? I guess we just need the current revision, WDYT?

Reply:
yes, we only need last revision for this

Reply:
Resolved with b684e00